tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -211,7 +211,7 @@ public:
    RColumnReaderBase *GetDatasetColumnReader(unsigned int slot, const std::string &col, const std::type_info &ti) const;
 
    /// End of recursive chain of calls, does nothing
-   void AddFilterName(std::vector<std::string> &) {}
+   void AddFilterName(std::vector<std::string> &) override {}
```

Comment:
why can this not be `final`? there should be no type inheriting from `RLoopManager`